src/pages/flow.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React from 'react'
```

Comment:
Sorry about the formatting noise, but I used the formatting from the included .prettierrc file in the project